### PR TITLE
Added Shared Dictionary

### PR DESCRIPTION
**Pull request by AgusHim**
Added shared dictionary to easy maintenance all static String

### CODE REVIEW

File: lib/shared/dictionary.dart [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const appname= 'My Meals App';
+const addFavoritesMsg= "Start adding some favorites ";
+const yourFilter="Your Filters";
+const yourFavorites="Your Favorites";
+const categories ='Categories';
```
Comment (Monik09) [inline lib/shared/dictionary.dart]: @Agushm  Add a new line here. after line 5.

Comment (AgusHim) [reply lib/shared/dictionary.dart]: Ok wait

File: lib/shared/dictionary.dart [hunk 2]
```diff
@@ -1,5 +1,5 @@
 const appname= 'My Meals App';
-const addFavoritesMsg= "Start adding some favorites ";
-const yourFilter="Your Filters";
-const yourFavorites="Your Favorites";
-const categories ='Categories';
\ No newline at end of file
+const addFavoritesMsg= 'Start adding some favorites ';
+const yourFilter='Your Filters';
+const yourFavorites='Your Favorites';
+const categories ='Categories';
```
Comment (Monik09) [inline lib/shared/dictionary.dart:5]: @Agushm I askded to have a blank empty line at the end of the file i.e. after line 5. Just press enter after line 5.

Comment (AgusHim) [reply lib/shared/dictionary.dart:5]: I have made it like that

Comment (AgusHim) [reply lib/shared/dictionary.dart:5]: in last commit

File: lib/shared/dictionary.dart [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const appname= 'My Meals App';
+const addFavoritesMsg= 'Start adding some favorites ';
+const yourFilter='Your Filters';
+const yourFavorites='Your Favorites';
+const categories ='Categories';
+
+
+
+
```
Comment (Monik09) [inline lib/shared/dictionary.dart]: i asked only 1 empty line. remove these extra empty lines

Comment (AgusHim) [reply lib/shared/dictionary.dart]: Sory sir, please check again

File: lib/shared/dictionary.dart [hunk 4]
```diff
@@ -3,7 +3,3 @@ const addFavoritesMsg= 'Start adding some favorites ';
 const yourFilter='Your Filters';
 const yourFavorites='Your Favorites';
 const categories ='Categories';
-
```
Comment (Monik09) [inline lib/shared/dictionary.dart]: 😞 see we only need 1 newline so just **press enter 1 time only**.

Comment (AgusHim) [reply lib/shared/dictionary.dart]: 6 line now. need one more?

Comment (Monik09) [reply lib/shared/dictionary.dart]: only 1 after line number 5

Comment (Monik09) [reply lib/shared/dictionary.dart]: no 6 lines

Comment (AgusHim) [reply lib/shared/dictionary.dart]: in my Text Editor is already 5 line and one line blank

Comment (Monik09) [reply lib/shared/dictionary.dart]: just have 1 more and then go check in your repo's file. Now there is no extra line at the end. you can recheck that

**Comment by Monik09** [comment]
LGTM 👍 🤩

**Pull request merged**